src/pages/Settings/PaymentSettings.vue

```diff
@@ -0,0 +1,339 @@
+<template>
+	<www-page class="payments" :gray-background="true">
+		<the-my-kiva-secondary-menu slot="secondary" />
+		<div class="title-area">
+			<div class="row column">
+				<h1>
+					Payments Settings
```

Comment:
This title and the title on the Settings index page should probably read the same. Overall, it seems awkward so I think the heading should be "Payment Method Settings" on both the settings index and the actual page.